application/src/test/java/org/opentripplanner/_support/time/DateTimes.java

```diff
@@ -0,0 +1,13 @@
+package org.opentripplanner._support.time;
+
+import java.time.LocalDate;
+import java.time.ZonedDateTime;
+
+public class DateTimes {
+
+  public static final ZonedDateTime ANY_ZONED_DATE_TIME_1 = ZonedDateTime.parse(
+    "2025-01-14T14:01:21+01:00"
+  );
+  public static final ZonedDateTime ANY_ZONED_DATE_TIME_2 = ANY_ZONED_DATE_TIME_1.plusHours(3);
```

Comment:
This does not make sense, if you can use any date in the test, then you can use date 1 and you do not need a second one.